### ISSUE TITLE
Installation Hangs Forever

### ISSUE THREAD

**Issue by wood-jp** [wbond/package_control#987]
I have a fresh installation of ST3, and have used both the copy-paste code and the manual way at https://packagecontrol.io/installation, but Sublime just hangs indefinitely.  When using the copy-paste code,  I get a print out of 186674 before it hangs. Also the content of Installed Packages changes to include:
> -rw-rw-r-- 1 jwood jwood   2315 Sep 14 23:46 0_package_control_loader.sublime-package
> -rw-rw-r-- 1 jwood jwood 186674 Sep 14 23:46 Package Control.sublime-package

OS: Ubuntu Mate 15.04
ST3: Build 3095 and Build 3083

Looking at my system monitor, I see that `sublime_text` and `plugin_host` are sleeping and waiting on `futex_wait_queue_me`

Killing Sublime and restarting has no effect.

**Comment by silentday** [comment]
Seeing the same thing. Looks like the settings file was blank on my install, tried copying settings from a machine that is working with no change.

**Comment by wbond** [comment]
@silentday Are you using Linux also?

**Comment by wbond** [comment]
I am thinking this is a duplicate of #988. What architecture are you running? x86_64 or i686?

**Comment by silentday** [comment]
Yep Ubuntu 15.04, x86_64.

**Comment by wbond** [comment]
Ok, closing this as a duplicate of #988 since that has more comments
